blocks/identity-block/components/Button/index.jsx

```diff
@@ -40,19 +42,41 @@ const StyledDynamicButton = styled.button.attrs((props) => ({
           background-color: #ffffff;
           border-color: #ffffff;
           color: ${getThemeStyle(arcSite)['primary-color']};
+
+          &:hover {
+            color: ${getThemeStyle(arcSite)['primary-color']};
+          }
         `;
       case BUTTON_STYLES.OUTLINED:
         return `
           background-color: transparent;
           border-color: ${getThemeStyle(arcSite)['primary-color']};
           color: ${getThemeStyle(arcSite)['primary-color']};
+
+          &:hover {
+            color: ${getThemeStyle(arcSite)['primary-color']};
+          }
+        `;
+      case BUTTON_STYLES.OUTLINED_GREY:
+        return `
+          background-color: transparent;
+          border-color: rgba(255, 255, 255, 0.5);
+          color: #ffffff;
+
+          &:hover {
+            color: #ffffff;
+          }
         `;
       case BUTTON_STYLES.FILLED:
       default:
         return `
           background-color: ${getThemeStyle(arcSite)['primary-color']};
           border-color: ${getThemeStyle(arcSite)['primary-color']};
```

Comment:
Why not pass in the primary color to avoid all the look ups?

Reply:
If only we had a styled component theme context :p

Reply:
updated. good refactor. also passed in font-family